nexus/blueprint-execution/tests/config.test.toml

```diff
@@ -0,0 +1 @@
+../../tests/config.test.toml
```

Comment:
Accidental addition maybe?

Reply:
Surprisingly, no.  The nexus-test-utils stuff loads a config file from a relative path and this enables it to work for this package.  omdb has a similar symlink.  (There may be a way to clean this up.)